### PR TITLE
fix bug where no auth crash home

### PR DESCRIPTION
**Pull request by mat10tng**

### CODE REVIEW

File: src/app/page/Home/Home.js [hunk 1]
```diff
@@ -382,7 +382,11 @@ const SignInHeaderComponent = ({ history }) => {
  */
 const HomePage = ({ auth, history, profile }) => {
   const org = useOrganization();
+
   useFirestoreConnect(() => {
+    if (!auth.uid) {
+      return [];
+    }
```
Comment (jwu910) [inline src/app/page/Home/Home.js:389]: Don't connect if no auth object is provided?

Comment (mat10tng) [reply src/app/page/Home/Home.js:389]: yes

Comment (mat10tng) [reply src/app/page/Home/Home.js:389]: I mean, its a hook, so we cant have an if outside of it

Comment (jwu910) [reply src/app/page/Home/Home.js:389]: :+1:

**Pull request merged**